bulbs/infographics/metadata.py

```diff
@@ -68,16 +66,17 @@ def get_serializer_info(self, serializer):
             ])
         return serializer_info
 
-    def get_custom_metadata(self, serializer, view):
+    def get_custom_metadata(self, serializer, view, list=False):
         fields_metadata = dict()
         if hasattr(serializer, "__call__"):
             serializer_instance = serializer()
         else:
             serializer_instance = serializer
         for field_name, field in serializer_instance.get_fields().items():
             if isinstance(field, InfographicDataField):
-                serializer = view.get_object().get_data_serializer()
-                fields_metadata[field_name] = self.get_custom_metadata(serializer, view)
+                if view.get_view_name() != "Content List":
```

Comment:
The only alternative I'm seeing atm is

```view.suffix == "List"``` unless we override the options, since we are trying to verify the origin of the request before the serializer is instantiated.